README.md

```diff
@@ -8,6 +8,13 @@ Containers images with tools for developers 👨‍💻👩‍💻
 
 ### Red Hat Universal Base Image ([UBI](https://developers.redhat.com/articles/ubi-faq#)) based image ([quay.io/devfile/base-developer-image:ubi8-latest](https://quay.io/repository/devfile/base-developer-image/))
 
+Build with Docker buildkit:
+
+```bash
+$ cd base/ubi8
+$ DOCKER_BUILDKIT=1 docker image build --progress=plain -t quay.io/devfile/base-developer-image:ubi8-latest .
+```
+
```

Comment:
We are mixing how to use it (for final users) with how to build it (for contributors). I think we should move the build instructions in a CONTRIBUTING.md file (like https://github.com/kubernetes/kubernetes-template-project/blob/main/CONTRIBUTING.md).

Reply:
Done